src/containers/Sensors/Sensor.tsx

```diff
@@ -107,12 +110,18 @@ function Sensor() {
     setQuestionIndex(questionIndex + 1);
     setAnswers(answers.concat(answer));
 
-    trackEvent({
-      category: `Feedback for ${sensor.data?.name} sensor`,
+    addFeedback({
```

Comment:
I assume it's intentional that we don't track network state of this call? We give the user no feedback that network call is being made, if there's an error, etc.

It's totally fine to not show feedback, if we're ok with losing this data, like with analytics if you don't catch 100% of the events it's not a big deal so you would never show any network activity. Just want to make sure we're being intentional.

Right now if there was an error, nothing is catching it (it gets rethrown in `addFeedback`) which could cause JS to stop running